### DISCUSSION TITLE
请问各位，现在如何在线安装Bilibili Evolved的各种组件？？？

### DISCUSSION THREAD

**Discussion by Meloetta-Nahida** [the1812/Bilibili-Evolved#5005] [提问 / Q&A]
请问各位，现在如何在线安装Bilibili Evolved的各种组件？？？在线仓库里一片空白，“空空如也”，还经常跳出“错误”弹窗，这让人如何安装组件？？？

![Bilibili Evolved如何安装2](https://github.com/user-attachments/assets/a66f7e09-d4e0-4d62-95ab-080d89c991a9)

![Bilibili Evolved如何安装3](https://github.com/user-attachments/assets/eba563a5-b1b3-40fa-a126-fa5d3afc4da1)

![Bilibili Evolved如何安装](https://github.com/user-attachments/assets/9b8020fd-a3f8-43a5-8493-d2ee2b8643cb)

**Comment by Geeyun-JY3** [comment]
请认真阅读 README：https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85

> **注意事项**
> 
> - ⚠ 使用正式版 (GitHub 源) 和预览版须翻墙.

**Reply by Meloetta-Nahida** [reply]
> 请认真阅读 README：https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85
> 
> > **注意事项**
> > 
> > * ⚠ 使用正式版 (GitHub 源) 和预览版须翻墙.

我知道了，刚才一段时间，Github的服务器一定出故障了，打开这个网页也是时好时坏的，也无法安装Bilibili EvolvedV2.9.4正式版和预览版，我这个帖子也发了好一段时间才发出来。。。

**Reply by Geeyun-JY3** [reply]
> 我知道了，刚才一段时间，Github的服务器一定出故障了，打开这个网页也是时好时坏的，也无法安装Bilibili EvolvedV2.9.4正式版和预览版，我这个帖子也发了好一段时间才发出来。。。

你的网络条件确定能正常使用 GitHub 吗？

**Reply by Meloetta-Nahida** [reply]
> > 我知道了，刚才一段时间，Github的服务器一定出故障了，打开这个网页也是时好时坏的，也无法安装Bilibili EvolvedV2.9.4正式版和预览版，我这个帖子也发了好一段时间才发出来。。。
> 
> 你的网络条件确定能正常使用 GitHub 吗？

可以正常使用GitHub。我挂了梯子。。。
